src/modules/storage/LedgerStorage.ts

```diff
@@ -518,4 +533,51 @@ export class LedgerStorage extends Storages
 
         this.query(sql, [], onSuccess, onError);
     }
+
+    /**
+     * Puts a block to database
```

Comment:
It would be better to explain the height of the block successfully stored.

Reply:
I changed it